src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -170,38 +276,42 @@ def test_WINNF_FT_S_WDB_2(self, config_filename):
     # Load the configuration file
     config = loadConfig(config_filename)
 
+    # Very light checking of the config file.
+    self.assertNotEqual(config['registrationRequests'], None)
```

Comment:
Huh? I think you want to verify that the length is > 0.

Reply:
Updated   as per the comment